utils/src/command_output.rs

```diff
@@ -1,12 +1,63 @@
 use std::{
     ffi::OsStr,
     fmt::Debug,
-    io::{Error, ErrorKind, Result},
-    process::{Command, Stdio},
+    io::{BufRead, BufReader, Error, ErrorKind, Result},
+    process::{Command, ExitStatus, Stdio},
+    thread,
     time::{Duration, Instant},
 };
 
+use colored::Colorize;
 use process_control::{ChildExt, Control};
+use rand::Rng;
+
+pub trait CommandExt {
+    /// Prints each line of stdout with a prefix string
+    /// that is given a random color.
+    ///
+    /// # Errors
+    /// Will error if there was an issue executing the process.
+    fn status_log_prefix<T: AsRef<str>>(&mut self, log_prefix: &T) -> Result<ExitStatus>;
+}
+
+impl CommandExt for Command {
+    fn status_log_prefix<T: AsRef<str>>(&mut self, log_prefix: &T) -> Result<ExitStatus> {
+        let mut child = self.stdout(Stdio::piped()).stderr(Stdio::piped()).spawn()?;
+
+        let mut rng = rand::thread_rng();
+        let red: u8 = rng.gen_range(80..=240);
+        let green: u8 = rng.gen_range(80..=240);
+        let blue: u8 = rng.gen_range(80..=240);
```

Comment:
@xynydev I could use your artistic eye in getting a good range of colors here. I just picked a range that wouldn't create something too light that you couldn't distinguish between the builds and something too dark for dark terminal users (such as myself).

Reply:
Could you maybe tell me where this is used and/or send a screenie?

Reply:
Also, for random color generation, setting a standard saturation and lightness, while randomizing the hue, is what I think could work best. Generative art, eh?

Reply:
Yeah this will be used for command output for builds. It will prepend each line with the image that is being built so that users can follow along easier. Let me get some examples

Reply:
![image](https://github.com/blue-build/cli/assets/4626052/0d587a8a-6530-47f5-9714-c14aa7cda13e)

Reply:
Here's one from a different terminal 
![image](https://github.com/blue-build/cli/assets/4626052/0d2326a9-9b9c-4529-baca-eb0afc743b6a)

Reply:
> Also, for random color generation, setting a standard saturation and lightness, while randomizing the hue, is what I think could work best. Generative art, eh?

I'll look into this. The library I use does RGB, but I know I could do some math for HSL.